docs/es/week04/04-1.md

```diff
@@ -111,7 +111,7 @@ La salida es la suma ponderada de las columnas de la matriz $\boldsymbol{A}$. Po
 
 <!-- Now we extend linear algebra to convolutions, by using the example of audio data analysis. We start with representing a fully connected layer as a form of matrix multiplication: -
 -->
-Ahora extenderemos el álgebra lineal a convoluciones, usando un ejemplo de análisis de datos de audio. Empezamos representando una capa fully connected en forma de matriz de multiplicación: -
+Ahora extenderemos el álgebra lineal a convoluciones, usando un ejemplo de análisis de datos de audio. Empezamos representando una capa densa (*fully connected*) como una matriz de multiplicación:
```

Comment:
```suggestion
Ahora extenderemos el álgebra lineal a convoluciones, usando un ejemplo de análisis de datos de audio. Empezamos representando una capa totalmente conectada como una matriz de multiplicación:
```
"Dense" (as opposed to "fully connected") is a *kind* of layer (e.g. another one is "convolutional").

 [The wiki](https://github.com/Atcold/pytorch-Deep-Learning/wiki/Spanish-translation) has "fully connected" as "capa totalmente conectada".